### ISSUE TITLE
neural network doesn't recognized own handwriting

### ISSUE THREAD

**Issue by Gerrit02** [makeyourownneuralnetwork/makeyourownneuralnetwork#34]
Hello, (in below example for number 2 and 3)
in attach, you can see the original pictures I took with my smartpone (1512x1512 pixels).
Then I converted them to 28x28 pixels and saved in png format.

But as you can see the network doesn't guess the correct label :-(
I trained the network with the training_set (60000 examples).

As you can also see the 28x28 pixels figures are more flew but still you can correctly see the contours of the numbers.
Maybe when hou converte from 1512x1512 to 28x28 there's become to much noice arround the contours?

Is that the reason my network doesn't recognized the numbers? Is there a way to filter out the noise or can making pictures in less quality help?

Thanks for the answer,

[
![original](https://user-images.githubusercontent.com/34437030/36659522-0f2f8c7e-1ad5-11e8-8b2e-4ddbdd01fe21.JPG)
![reducedimage](https://user-images.githubusercontent.com/34437030/36659523-0f4b0c4c-1ad5-11e8-8b46-0e958a6389e0.JPG)

](url)

**Comment by makeyourownneuralnetwork** [comment]
you are correct - the training data set has been "normalised"

that means the digits have been rescaled to a similar size to each other, centred too.

the colours are greyscale - so you'll need a well lit scan / photo. I tested with a bitmap editor.
